java-manta-client/src/main/java/com/joyent/manta/http/EncryptionHttpHelper.java

```diff
@@ -411,9 +411,18 @@ public MantaObjectInputStream httpRequestAsInputStream(final HttpUriRequest requ
             binaryEndPositionInclusive = ciphertextSize;
         // This is the typical case like: bytes=3-44
         } else {
+
+            long scaledPlaintextEnd = plaintextEnd;
+
+            // interpret maximum plaintext value as unbounded end
+            if (plaintextEnd == cipherDetails.getMaximumPlaintextSizeInBytes()) {
+
```

Comment:
I usually leave no space if there aren't brackets, and space if there are brackets (my thoughts were that visually it's easier to notice the distinct code block reading the code).  I'm definitely not set on a specific style though.